assets/locales/en.json

```diff
@@ -541,7 +541,8 @@
     "skip": "Skip",
     "continue": "Continue",
     "retry": "Try again",
-    "error": "Something went wrong"
+    "error": "Something went wrong",
+    "end_of_list": "You have reached the end of the list"
```

Comment:
The only remark I have is that `There are no logged activities yet.` ends with a period and `You have reached the end of the list` doesn't. That feels a bit inconsistent. Furthermore, everything looks fine.